pipelines/sqre/validate_drp_gen3.groovy

```diff
@@ -315,11 +315,10 @@ def void buildDrpGen3(Map p) {
       source /opt/lsst/software/stack/loadLSST.bash
       setup -k -r .
       
-      pip install --user dustmaps
-      python -c "import dustmaps.sfd;dustmaps.sfd.fetch()"
+      git clone https://github.com/lsst/dustmaps_cachedata
+      setup -k -r dustmaps_cachedata
 
       set -o xtrace
-      export PYTHONPATH=$PYTHONPATH:$HOME/.local/lib/python3.7/site-packages/
```

Comment:
what's this removal doing? Old cruft? (the commit message doesn't say).

Reply:
Sorry.  Yes, I was assuming that was there to pick up the `dustmaps` installed with `pip install --user dustmaps`.  I think it's broken anyway since we are on python 3.8 now